src/module.js

```diff
@@ -124,106 +130,151 @@ function createInitialHalsteadItemState () {
     };
 }
 
-function processLloc (node, syntax, currentReport) {
-    incrementCounter(node, syntax, 'lloc', incrementLogicalSloc, currentReport);
-}
-
-function incrementCounter (node, syntax, name, incrementFn, currentReport) {
-    var amount = syntax[name];
-
-    if (check.number(amount)) {
-        incrementFn(currentReport, amount);
-    } else if (check.function(amount)) {
-        incrementFn(currentReport, amount(node));
+/**
+ * check if amount a function, then call it with the given node and return a number
+ * @param amount
+ * @param node
+ * @returns {*}
+ */
+function convertToNumber(amount, node) {
+    if (check.function(amount)) {
+        amount = amount(node);
     }
-}
-
-function incrementLogicalSloc (currentReport, amount) {
-    debug('incrementing sloc by ' + amount);
-    report.aggregate.sloc.logical += amount;
-
-    if (currentReport) {
-        currentReport.sloc.logical += amount;
+    if (!check.number(amount)) {
```

Comment:
It is more direct to express conditional blocks with the positive case first. Please do so where possible.

e.g.
```
if (check.number(amount)) {
  return amount;
}
return 0;
```